vllm/core/scheduler.py

```diff
@@ -381,6 +381,19 @@ def abort_seq_group(self, request_id: Union[str, Iterable[str]]) -> None:
                     seq.status = SequenceStatus.FINISHED_ABORTED
                     self.free_seq(seq)
 
+                self._free_seq_group(aborted_group)
+
+    def _free_seq_group(
+        self,
+        seq_group: SequenceGroup,
+    ) -> None:
+        """
+        Free a sequence group from a cross-attention block table.
```

Comment:
I think the function's name doesn't describe what it does clearly enough. Perhaps something like `_free_cross_attn_blocks` or so

Reply:
Good point. The seq group identifies where to perform the free operation, however the free operation actually targets the cross-attn blocks that are stored at the seq group level. Thus,

_free_seq_group() -> _free_seq_group_cross_attn_blocks()

in the latest push.